hs_core/templates/pages/genericresource.html

```diff
@@ -2577,6 +2620,11 @@ <h3 style="color: red" id="file-type-error"></h3>
                             $form.parent().closest(".user-roles").find("li").removeClass("loading");
                             setPointerEvents(true);
                         }
+                    },
+                    error: function(XMLHttpRequest, textStatus, errorThrown){
+                        $("#div-invite-people").find(".label-danger").remove(); // Remove previous alerts
```

Comment:
Same comment as above to make sure find() does not return null when calling remove() on it.